zulipterminal/ui_tools/boxes.py

```diff
@@ -724,8 +727,10 @@ def keypress(self, size: urwid_Size, key: str) -> Optional[str]:
         elif is_command_key('TOGGLE_NARROW', key):
             self.model.unset_search_narrow()
             if self.message['type'] == 'private':
-                if (len(self.model.narrow) == 1 and
-                        self.model.narrow[0][0] == 'pm_with'):
+                if (
+                    len(self.model.narrow) == 1
+                    and self.model.narrow[0][0] == 'pm_with'
+                   ):
```

Comment:
A compact alternative could be:

```python
if len(self.model.narrow) == 1 \
   and self.model.narrow[0][0] == 'pm_with':
```

Reply:
Right - I'm trying to avoid continuation characters.

I'm not fond of the single-line parentheses, but I do think they're cleaner.

Also see https://www.python.org/dev/peps/pep-0008/#maximum-line-length